net-dialup/xl2tpd/xl2tpd-1.3.12.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd toolchain-funcs
+
+DESCRIPTION="A modern version of the Layer 2 Tunneling Protocol (L2TP) daemon"
+HOMEPAGE="http://www.xelerance.com/services/software/xl2tpd/"
+SRC_URI="https://github.com/xelerance/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~ppc64 ~x86"
+IUSE="dnsretry"
+
+DEPEND="
+	net-libs/libpcap
+	>sys-kernel/linux-headers-2.6"
+
+RDEPEND="
+	${DEPEND}
+	net-dialup/ppp"
+
+DOCS=(CREDITS README.xl2tpd BUGS CHANGES TODO doc/README.patents)
+
+src_prepare() {
+	default
+	# The below patch is questionable. Why wasn't it submitted upstream? If it
+	# ever breaks, it will just be removed. -darkside 20120914
+	# Patch has been discussed with upstream and is marked as feature by now:
+	# https://github.com/xelerance/xl2tpd/issues/134 // -- tenX 2017-08-06
+	use dnsretry && eapply -p0 "${FILESDIR}/${PN}-dnsretry.patch"
```

Comment:
why are you trying to keep ugly strip level modification rather thab modify the patch?

Reply:
Please do not act as if this has been just introduced, it has been in ebuilds since xl2tpd-1.3.1-r2 and possibly even before that. When I took over maintainership, I really tried to clear up the background of the patch and possibly get it integrated, you for sure read up on this already: [https://github.com/xelerance/xl2tpd/issues/134](https://github.com/xelerance/xl2tpd/issues/134)
Since it is unclear to me who still needs this I did not think it was a good idea to change the conditional nature of it. If it has become unacceptable by now we should finally kick it out completely.

Reply:
@Zlogene: That is of course fair and important. I did not really like the construct from the beginning but can see some value in the patch itself. Upstream does not really so far. It is only relevant for xl2tpd client usage in a specific scenario when probably most people use it in server mode. How about leaving it like it is for now and I will ping them on the issue again? If there are no changes up to the next version of xl2tpd we can kick it out. 
The next step after this version bump to 1.3.12 will be full stabilization of 1.3.10 and the elimination of all older versions.